### DISCUSSION TITLE
Porting Blue kitchen to non-ESP32

### DISCUSSION THREAD

**Discussion by rsemenoff** [pschatzmann/ESP32-A2DP#438] [Q&A] [answered]
Hi,  I just discovered ESP32-A2DP last week, tried the examples, and ..success! 
But then, something stopped working and I suspect my cheap esp32 board is flaking out
(maybe a consequence of the esp32 supply chain?), so I thought I'd see if I could get
A2DP for my pico W (which has been quite reliable for me)  as it seems (according to Blue Kitchen)
to be available, just not as an arduino library.  I suppose I could do that myself, but I'd trust your
library development skills more than my own (to be honest). So I'm wondering if you have already
tried making A2DP work on pico W and failed ?  or was it not your intention to do that (ever, or at this time)  ? Thanks!
BTW, the audio library looks really cool as well , going to look into using it to do filtering of sonar/radar sensor data for yet another future project ...

**Comment by pschatzmann** [comment, marked as answer]
I started to [work on it](https://github.com/pschatzmann/RP2040-A2DP)  but then run into medical issues.
Maybe you want to pick this up

**Comment by rsemenoff** [comment]
Thank you for this, but also, hope you are taking sufficient rest from it !I will check it out ..
    On Tuesday, July 18, 2023 at 11:44:08 PM PDT, Phil Schatzmann ***@***.***> wrote:  
 
 


I started to work on it but then run into medical issues.
Maybe you want to pick this up
